src/tiled/commanddatamodel.cpp

```diff
@@ -250,7 +257,9 @@ Qt::ItemFlags CommandDataModel::flags(const QModelIndex &index) const
     Qt::ItemFlags f = QAbstractTableModel::flags(index);
 
     if (isNormalRow) {
-        f |= Qt::ItemIsDragEnabled | Qt::ItemIsDropEnabled;
+        // no reasons to support support 'Drag&Drop' for working dir
+        if (index.column() != WorkingDirColumn)
+            f |= Qt::ItemIsDragEnabled | Qt::ItemIsDropEnabled;
```

Comment:
Why this? I think it's a strange omission and would rather see `dropMimeData` be fixed to pass the working directory to the copy.